queries.yaml

```diff
@@ -1712,19 +1712,20 @@ post_process:
   # drop small inners in buildings before we do the merge.
   - fn: vectordatasource.transform.drop_small_inners
     params:
-      end_zoom: 15
+      end_zoom: 17
       source_layers: [buildings]
-      pixel_area: 0.25
+      pixel_area: 10.0
 
   - fn: vectordatasource.transform.quantize_height
     params:
       source_layer: buildings
       start_zoom: 13
-      end_zoom: 16
+      end_zoom: 17
       quantize:
         13: vectordatasource.transform.quantize_height_round_nearest_20_meters
         14: vectordatasource.transform.quantize_height_round_nearest_10_meters
         15: vectordatasource.transform.quantize_height_round_nearest_10_meters
+        16: vectordatasource.transform.quantize_height_round_nearest_5_meters
```

Comment:
This appear to be quantizing a 512 px coordinate zoom 16 tile in error (like in 16/19298/24630.mvt).

Like https://www.openstreetmap.org/way/290539412 with a height of 136 is exported as 135 instead.

<img width="599" alt="image" src="https://user-images.githubusercontent.com/853051/153310431-57d826cf-f684-4311-8cc4-081f6a0e4efe.png">

Reply:
Given the overall height of the building, and how many pixels are involved, this might be fine.

Reply:
made a change on tilequeue let's see whether that can fix it: https://github.com/tilezen/tilequeue/pull/407/

Reply:
Confirmed Tilequeue changes result in expected height quantification at zoom 16 (none) in 512px space and at zoom 15 (rounded to 5)